includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -37,6 +38,36 @@ public static function onFetchTemplateAndTitle( &$text, &$finalTitle ) {
 	}
 
 	/**
+	 *
+	 * @param string $text - template content
+	 * @param Title $finalTitle - template title object
+	 *
+	 * @return bool
+	 */
+	public static function onFetchTemplateAndTitleForTables( &$text, &$finalTitle ) {
+		wfProfileIn( __METHOD__ );
+		// do wiki tables markings
+		$text = DataTables::markTranscludedTables( $text );
+
+		wfProfileOut( __METHOD__ );
+
+		return true;
+	}
+
+	/**
+	 * @param $parser
+	 * @param $text
+	 * @param $stripState
+	 *
+	 * @return bool
+	 */
+	public static function onAfterDoTableStuff( &$parser, &$text, &$stripState ) {
+		$text = DataTables::markDataTables( $text );
```

Comment:
Please add new `wg` flag for this feature sot it could be easily turned on/off for each community
 Add a line note